lib/DDG/Util/SpiceConstants.pm

```diff
@@ -0,0 +1,9 @@
+package DDG::Util::SpiceConstants;
+
+use strict;
+use warnings;
+
+# regex for allowed TLDS (grabbed from DDG core repo, in /lib/DDG/Util/Constants.pm)
```

Comment:
Original comment in `IsItUp.pm` 
said that. Didn't break into your servers for a regex, :stuck_out_tongue_winking_eye: